pressbooks/pressbooks-book
# Avoid width css rule for PDF in images for buckram package

https://github.com/pressbooks/pressbooks-book/issues/699
 ### How to test
- Place an image with a caption without any alignment in any chapter
- Change to one of those themes: McLuhan, Malala or Jacobs
- Export to PDF and check image captions not aligned stays on the bottom of picture.
